### PR TITLE
[BACKLOG-5840] -- added function to store default block ui options an…

### PR DESCRIPTION
**Pull request by krivera-pentaho**
…d to use them

### CODE REVIEW

**Comment by krivera-pentaho** [comment]
@diogofscmariano @jvelasques @plagoa @pamval @annapopova1 please review

File: core-js/src/main/javascript/cdf/dashboard/Dashboard.notifications.js [hunk 1]
```diff
@@ -55,6 +55,10 @@ define([
       };
     },
 
+    _setBlockUiOptions: function(options) {
```
Comment (pamval) [inline core-js/src/main/javascript/cdf/dashboard/Dashboard.notifications.js]: Documentation for this function ?

**Comment by diogofscmariano** [comment]
Will this not override this? https://github.com/webdetails/cdf/blob/master/core-js/src/main/javascript/cdf/dashboard/Dashboard.js#L144
Also, i would like to have this code probably executed using that function. Since we are adding something on the cdf side lets do it in the right way

**Comment by krivera-pentaho** [comment]
@diogofscmariano @pamval I updated my pull request. Please review

**Pull request merged**